home.md

```diff
@@ -1,24 +1,57 @@
-Changing the provider will update all of the information shown within the inputs
-application. You will immediately see the map update showing the selected
-provider, as well as that trusts peers, as defined by the
-[Trust Peer Finder Tool][1]. The list of these peers is displayed below the
-map (collapsed by default). Clicking on any of the points on the map will reveal
-the name of the provider.
+---
+editor_options: 
+  markdown: 
+    wrap: 72
```

Comment:
Awkwardly, this editor setting has changed almost every line of this file so that they wrap at 72 characters.